src/components/editor/Bindings/ManualBackfill.tsx

```diff
@@ -0,0 +1,149 @@
+                : false,
+        [backfilledCollections, currentCollection]
+    );
+
+    const serverUpdateRequired = useMemo(() => {
+        if (currentCollection && increment !== 'undefined') {
+            return increment === 'true'
+                ? !backfilledCollections.includes(currentCollection)
+                : backfilledCollections.includes(currentCollection);
+        }
+
+        return false;
+    }, [backfilledCollections, currentCollection, increment]);
+
+    const draftSpec = useMemo(
+        () =>
+            draftSpecs.length > 0 && draftSpecs[0].spec ? draftSpecs[0] : null,
+        [draftSpecs]
+    );
+
+    useEffect(() => {
+        if (
+            draftSpec &&
+            currentCollection &&
+            serverUpdateRequired &&
+            increment !== 'undefined'
+        ) {
+            setFormState({ status: FormStatus.UPDATING });
+
+            updateBackfillCounter(
+                draftSpec,
+                bindingIndex,
+                increment,
+                currentCollection
+            ).then(
+                () => {
+                    increment === 'true'
+                        ? addBackfilledCollection(currentCollection)
+                        : removeBackfilledCollection(currentCollection);
+
+                    setFormState({ status: FormStatus.UPDATED });
+                },
+                (error) =>
+                    setFormState({
+                        status: FormStatus.FAILED,
+                        error: {
+                            title: 'workflows.collectionSelector.manualBackfill.error.title',
+                            error,
+                        },
+                    })
+            );
+        }
+    }, [
+        addBackfilledCollection,
+        bindingIndex,
+        currentCollection,
+        draftSpec,
+        increment,
+        removeBackfilledCollection,
+        serverUpdateRequired,
+        setFormState,
+        updateBackfillCounter,
+    ]);
+
+    return (
+        <Box sx={{ mt: 3 }}>
+            <Stack direction="row" spacing={1} sx={{ mb: 2 }}>
+                <Stack spacing={1}>
+                    <Typography variant="h6" sx={{ mr: 0.5 }}>
+                        <FormattedMessage id="workflows.collectionSelector.manualBackfill.header" />
+                    </Typography>
+
+                    <Typography component="div">
+                        <FormattedMessage
+                            id={`workflows.collectionSelector.manualBackfill.message.${entityType}`}
+                        />
+                    </Typography>
+                </Stack>
+            </Stack>
+
+            <OutlinedToggleButton
+                value={increment}
+                selected={selected}
+                disabled={formActive}
+                onClick={(event, checked: string) => {
+                    event.preventDefault();
+                    event.stopPropagation();
```

Comment:
I don't think we need there here as I don't think the container isn't listening for a click.